### PR TITLE
travis: Push releases to Jenkins SCP mirror

### PR DESCRIPTION
**Pull request by kklin**
Before, Jenkins pulled the GitHub releases for every test run. This led
to inflated download metrics. This commit pushes releases to a SCP
server in addition to GitHub so that Jenkins won't need to hit the
GitHub release when testing.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -39,16 +42,24 @@ notifications:
   slack:
     secure: jap4ODSrC9Nrd61tU3vSB9z8GBUUQmVcQzHWzIUHtQ2orGE8NqlrgdL17Bang26mm083vjFuaHb82OeJXeeMnH+8WFBY8ju19KJI/4Lgnou+Q0eOZ6bmHY9fjIgLTm+HiOrMaeDiDtLUPQIm1O4aNQ1GjT410LeQ0QkFZSPRJN+kxYDvLW8GQi3kG12BvIWPWFJY/r5B6JvSWQPzsJXvFTFAfu5B9enhwzy/s7W9tT+ddpnuCyz2+O2ADpGfLwtyH50jLlmIyrq3QKT6vYy2COq5GZ3yM2noaQRYRbl/5LIPdsAh99wjJF4X9oNXCl4EmxK7sWPKZc7Yk/AIGVHhlOlkkYhYzbajw3OPLo/02RDJv4IPCpFKfePOYLNpvcH4F+ae4ZQ1NmU0ksH9q04fTUH6xtwRU74WIiaelWMoH4qegUR4GJHbIEbCcT1HwN36ZkAeIFtY8HHT9+3wag1nnWG3l3uRIudcvjYFp7flGhaqW/t+VFbxaKRbahRPHV9K71XajdxR2GoQOggaE6k7/TeGBkHDR/3zX0sA30+meHxrb8Zo3py+Lsb8CQ5ULE/TWyGQyLN8RN/ynj+qtvI6nVIgKzkD+pgluerEIu5KhA0WslHY1Z0cQgI8mmoU4SY5Tzim/kwyljmBVFAIafyDibc5OjVSRpGca1vtBc18ZrY=
 
-before_deploy: make release
+before_deploy:
+    - make release
+    - openssl aes-256-cbc -K $encrypted_e63862ef6e81_key -iv $encrypted_e63862ef6e81_iv
```
Comment (kayousterhout) [inline .travis.yml]: What is this doing? (can you add a comment?)

File: scripts/scp-to-jenkins [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```
Comment (kayousterhout) [inline scripts/scp-to-jenkins]: Can you write a brief comment at the top of this file saying what this is doing?

Comment (kklin) [reply scripts/scp-to-jenkins]: You meant saying what this script is doing right? Not the shebang?

Comment (kayousterhout) [reply scripts/scp-to-jenkins]: Yup!

File: .travis.yml [hunk 3]
```diff
@@ -1,5 +1,8 @@
 language: go
 
+env:
+    - RELEASER_KEY_PATH=./scripts/releaser_key
```
Comment (kayousterhout) [inline .travis.yml]: Can you add one more comment here -- saying this is a key that has been encrypted using Travis's secret key?

**Comment by kklin** [comment]
Note to self -- this shouldn't be merged until the SCP server is deployed or else the build will fail.

**Pull request merged**